std/experimental/logger/core.d

```diff
@@ -1801,6 +1801,13 @@ unittest
     stdThreadLocalLog = new FileLogger("someFile.log");
 
     stdThreadLocalLog = l;
+
+    scope(exit)
+    {
+        import std.file : exists;
```

Comment:
He was using it in an assert to check the file was removed, but it wasn't working on Windows, so he removed it but forgot to remove the import.